src/plugin/mod.rs

```diff
@@ -1,6 +1,7 @@
 pub use self::configuration::{RapierConfiguration, SimulationToRenderTime, TimestepMode};
 pub use self::context::RapierContext;
 pub use self::plugin::{NoUserData, PhysicsSet, RapierPhysicsPlugin, RapierTransformPropagateSet};
+pub use narrow_phase::ContactPairView;
```

Comment:
Should we add more ? `ContactManifoldView`, `SolverContactView`, `ContactView` ?

Reply:
Sounds like a good idea, yes.